tests/firedrake/regression/test_bddc.py

```diff
@@ -120,11 +135,34 @@ def mesh(request):
 
 
 @pytest.mark.parallel
+@pytest.mark.parametrize("degree", range(1, 3))
+@pytest.mark.parametrize("variant", ("spectral", "fdm"))
+def test_vertex_dofs(mesh, variant, degree):
```

Comment:
Please can you add a quick docstring? I'm not familiar with BDDC so it isn't clear to me why I should expect this test to pass or fail.